os_net_config/common.py

```diff
@@ -272,7 +272,7 @@ def _get_dpdk_mac_address(name):
 
 def interface_mac(name):
     try:  # If the iface is part of a Linux bond, the real MAC is only here.
-        with open(get_dev_path(name, 'bonding_slave/perm_hwaddr'),
+        with open(get_dev_path(name, '_bonding_slave/perm_hwaddr'),
```

Comment:
Checked in latest rhel9.4, the path hasn't changed:
[cloud-admin@compute-0 ~]$ sudo cat /sys/class/net/enp130s0f0/bonding_slave/perm_hwaddr
3c:fd:fe:33:a8:a0
[cloud-admin@compute-0 ~]$ sudo cat /sys/class/net/enp130s0f1/bonding_slave/perm_hwaddr
3c:fd:fe:33:a8:a2
[cloud-admin@compute-0 ~]$ cat /etc/redhat-release 
Red Hat Enterprise Linux release 9.4 (Plow)
[cloud-admin@compute-0 ~]$ 

Can we confirm this ?

Reply:
Hi Viji,
get_dev_path(enp130s0f1, '_bonding_slave/perm_hwaddr') will give  /sys/class/net/enp130s0f1/bonding_slave/perm_hwaddr

while get_dev_path(enp130s0f1, 'bonding_slave/perm_hwaddr') will return /sys/class/net/enp130s0f1/device/bonding_slave/perm_hwaddr which is incorrect.